Formula/g/gabo.rb

```diff
@@ -0,0 +1,20 @@
+class Gabo < Formula
+  desc "Generates GitHub Actions boilerplate"
+  homepage "https://github.com/ashishb/gabo"
+  url "https://github.com/ashishb/gabo/archive/refs/tags/v1.1.2.tar.gz"
+  sha256 "2aba8f85f9d241217a56847c372f7d7a001218073f76f8e791bc161edd525611"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    cd "src/gabo" do
+      system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/gabo"
+    end
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w"), ".src/gabo/cmd/gabo"
```
Does this not work?

Reply:
that wont work as go.mod is in the `src/gabo`